src/components/Tool/Appeal/List/AppealTour/AppealTour.tsx

```diff
@@ -0,0 +1,233 @@
+    zIndex: '50',
+    background: color,
+    top: '50%',
+    transform: 'translateY(-50%) rotate(45deg)',
+  },
+  '@media (max-width: 550px)': {
+    width: '100%',
+    padding: `${theme.spacing(1)} ${theme.spacing(1)}`,
+    '&:after': {
+      width: '16px',
+      height: '16px',
+      top: '100%',
+      right: '50%',
+      transform: 'translate(50%, -50%) rotate(45deg)',
+    },
+  },
+}));
+
+type ActionButtonsProps = {
+  nextTourStep: () => void;
+  hideTour: () => void;
+  tourStep: AppealTourEnum;
+  t: TFunction;
+};
+const ActionButtons = ({
+  nextTourStep,
+  hideTour,
+  tourStep,
+  t,
+}: ActionButtonsProps) => {
+  return (
+    <Box display={'flex'} sx={{ marginTop: theme.spacing(2) }}>
+      <Box>
+        {tourStep !== AppealTourEnum.Finish && (
+          <Button
+            onClick={() => {
+              hideTour();
+            }}
+            variant="outlined"
+            sx={{ marginRight: theme.spacing(2) }}
+          >
+            {t('Hide')}
+          </Button>
+        )}
+
+        <Button
+          onClick={() => {
+            nextTourStep();
+          }}
+          variant="contained"
+        >
+          {tourStep === AppealTourEnum.Start && t('Start')}
+          {tourStep === AppealTourEnum.Finish && t('Finished')}
+          {tourStep === AppealTourEnum.ExportContacts && t('Export to CSV')}
+
+          {tourStep !== AppealTourEnum.Start &&
+            tourStep !== AppealTourEnum.ExportContacts &&
+            tourStep !== AppealTourEnum.Finish &&
+            t('Next')}
+        </Button>
+      </Box>
+    </Box>
+  );
+};
+
+type StatisticContainerProps = {
+  color: string;
+  statisticNumber: number;
+  statisticName: string;
+};
+const StatisticContainer = ({
+  color,
+  statisticNumber,
+  statisticName,
+}: StatisticContainerProps) => {
+  return (
+    <StatisticBox color={color}>
+      <Typography
+        variant="h4"
+        fontWeight="bold"
+        color={theme.palette.common.white}
+      >
+        {statisticNumber ?? 0}
+      </Typography>
+      <Typography
+        variant="body1"
+        fontWeight="bold"
+        color={theme.palette.common.white}
+      >
+        {statisticName ?? ''}
+      </Typography>
+    </StatisticBox>
+  );
+};
+
+export const AppealTour: React.FC = () => {
+  const { t } = useTranslation();
+  const [tourTitle, setTourTitle] = React.useState<string>('');
+  const [tourText, setTourText] = React.useState<string>('');
+  const {
+    tour,
+    hideTour,
+    nextTourStep,
+    askedCountQuery,
+    excludedCountQuery,
+    committedCountQuery,
+    givenCountQuery,
+    receivedCountQuery,
+  } = React.useContext(AppealsContext) as AppealsType;
+
+  const { data: askedCount } = askedCountQuery;
+  const { data: excludedCount } = excludedCountQuery;
+  const { data: committedCount } = committedCountQuery;
+  const { data: givenCount } = givenCountQuery;
+  const { data: receivedCount } = receivedCountQuery;
+
+  useEffect(() => {
```

Comment:
Instead of calculating these with state and `useEffect`, let's make it declarative with `useMemo`.

Reply:
@canac I thought when you used useMemo, you had to return a value. Are you talking about it returning an object like 

```
{
   tourTitle: 'Title',
   tourText: 'Text'
}
```

Reply:
Exactly. I was thinking something like this:

```ts
  const { tourTitle, tourText } = useMemo(() => {
    switch (tour) {
      case AppealTourEnum.Start:
        return {
          tourTitle: t('Appeal created successfully'),
          tourText: t('Get started by reviewing your excluded contacts list.'),
        };
      case AppealTourEnum.ReviewExcluded:
        return {
          tourTitle: t('Review Excluded'),
          tourText: t(
            'Go through your list of excluded contacts to see if they should be asked',
          ),
        };
      // ...
      default: {
        return { tourTitle: '', tourText: '' };
      }
    }
  }, [tour]);
  ```

Reply:
Done